examples/intro_tutorial.ipynb

```diff
@@ -0,0 +1,596 @@
+   "metadata": {},
+   "source": [
+    "### Check Request validity\n",
+    "\n",
+    "Before submitting a Harmony Request, you can test your request to see if it's valid and how to fix it if not. In particular, `request.is_valid` will check to ensure that your spatial BBox bounds and temporal ranges are entered correctly.  "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(f\"Request valid? {request.is_valid()}\")\n",
+    "for m in request.error_messages():\n",
+    "    print(\" * \" + m)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Submit Request\n",
+    "\n",
+    "Once you have created your request, you can now submit it to Harmony using the Harmony Client object. A job id is returned, which is a unique identifier that represents your submitted request. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "job_id = harmony_client.submit(request)\n",
+    "job_id"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Check Request status\n",
+    "\n",
+    "You can check on the progress of a processing job with `status()`. This method blocks while communicating with the server but returns quickly. The JSON helper function provides a nicer formatting option for viewing the status message. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "JSON(harmony_client.status(job_id))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Depending on the size of the request, you may want to wait until the request has completed processing before the remainder of your code is executed. The `wait_for_processing()` method will block subsequent lines of code while optinally showing a progress bar."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "harmony_client.wait_for_processing(job_id, show_progress=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### View Harmony job response and output URLs\n",
+    "\n",
+    "Once your data request has finished processing, you can view details on the job that was submitted to Harmony, including the API call to Harmony, and informational messages on your request if available. \n",
+    "\n",
+    "`result_json()` calls `wait_for_processing()` and returns the complete job in JSON format once processing is complete. You may optionally show the progress bar using the same `show_progress=True` statement in the previous line. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data = harmony_client.result_json(job_id)\n",
```

Comment:
I'm getting a failure here - it's not your code, something on the Harmony side of things.

{"username":"rmwaters","status":"failed","message":"Failed","progress":0,"createdAt":"2021-04-03T00:46:46.727Z","updatedAt":"2021-04-03T00:47:04.138Z","links":[{"title":"Job Status","href":"https://harmony.earthdata.nasa.gov/jobs/7e68f4eb-3f63-4e6e-b5ac-9224992b8b2a","rel":"self","type":"application/json"}],"request":"https://harmony.earthdata.nasa.gov/SENTINEL-1_INTERFEROGRAMS/ogc-api-coverages/1.0.0/collections/science%2Fgrids%2Fdata%2FunwrappedPhase/coverage/rangeset?forceAsync=true&subset=lat(19.26%3A19.94)&subset=lon(-155.75%3A-155.3)&subset=time(%222020-02-24T00%3A00%3A00%22%3A%222020-02-25T00%3A00%3A00%22)&format=image%2Ftiff&maxResults=2","numInputGranules":2,"jobID":"7e68f4eb-3f63-4e6e-b5ac-9224992b8b2a"}

These errors make it difficult to test the notebook further right now.